contracts/external/WETH9.sol

```diff
@@ -0,0 +1,78 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
```

Comment:
Am I right in thinking this is purely for testing purposes?

If so could we use ERC20 from Openzeppelin instead of including this?

Reply:
Yes, this is solely for local testing purposes. If you check the migrations scripts, on those we connect to the existing deployments. We will never deploy `WETH9`. 

Now that `WETH9` extends `ERC20` with `deposit` and `withdraw` functions. 

The `WETH9` included is the same that is deployed in mainnet (and holds $2B). To make sure we test against the deployed bytecode, we must use exactly this contract.